mdpow/tests/test_solvation.py

```diff
@@ -20,10 +20,10 @@
 
 @pytest.fixture
 def setup(tmpdir):
+    os.system('echo')
     newdir = tmpdir.mkdir('resources')
     old_path = os.getcwd()
-    resources = os.path.join(
-        old_path, 'mdpow', 'tests', 'testing_resources')
+    resources = os.path.join(old_path, 'testing_resources')
```

Comment:
why path changing?

Reply:
was an issue with my testing config pathced out in 6779c73